Model training errors out with ValueError

As seen in this [job run](https://github.com/oindrillac/time-to-merge-tool/actions/runs/3487060464/jobs/5834249321), the model training steps fails on occasion with a Value Error when it encounters NaNs. Investigate this issue and add steps to handle this error.

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Input In [21], in <cell line: 1>()
----> 1 X = raw_data_processor.fit_transform(pr_df)

File /opt/app-root/lib64/python3.8/site-packages/sklearn/compose/_column_transformer.py:[67](https://github.com/oindrillac/time-to-merge-tool/actions/runs/3487060464/jobs/5834249321#step:7:68)3, in ColumnTransformer.fit_transform(self, X, y)
    670 self._validate_column_callables(X)
    671 self._validate_remainder(X)
--> 673 result = self._fit_transform(X,y,_fit_transform_one)
    675 if not result:
    676     self._update_fitted_transformers([])

File /opt/app-root/lib64/python3.8/site-packages/sklearn/compose/_column_transformer.py:604, in ColumnTransformer._fit_transform(self, X, y, func, fitted, column_as_strings)
    598 transformers = list(
    599     self._iter(
    600         fitted=fitted, replace_strings=True, column_as_strings=column_as_strings
    601     )
    602 )
    603 try:
--> 604     return Parallel(n_jobs=self.n_jobs)(
    605 delayed(func)(
    606 transformer=clone(trans)ifnotfittedelsetrans,
    607 X=_safe_indexing(X,column,axis=1),
    608 y=y,
    609 weight=weight,
    610 message_clsname="ColumnTransformer",
    611 message=self._log_message(name,idx,len(transformers)),
    612 )
    613 foridx,(name,trans,column,weight)inenumerate(transformers,1)
    614 )
    615 except ValueError as e:
    616     if "Expected 2D array, got 1D array instead" in str(e):

File /opt/app-root/lib64/python3.8/site-packages/sklearn/preprocessing/_encoders.py:126, in _BaseEncoder._fit(self, X, handle_unknown, force_all_finite, return_counts)
    121     if diff:
    122         msg = (
    123             "Found unknown categories {0} in column {1}"
    124             " during fit".format(diff, i)
    125         )
--> 126         raise ValueError(msg)
    127 if return_counts:
    128     category_counts.append(_get_counts(Xi, cats))

ValueError: Found unknown categories ['NaN'] in column 0 during fit
ValueError: Found unknown categories ['NaN'] in column 0 during fit

Error: Process completed with exit code 1.
```

--- a/process_pr.py
+++ b/process_pr.py
@@ -1,5 +1,5 @@
 # The pre-processing used below is based on https://github.com/thoth-station/ttm-as-a-service 
-# and it is authored by Gage Krumbach
+# and it is originally authored by Harshad Reddy Nalla
 
 from github import Github
 from github.PullRequest import PullRequest as GithubPullRequest
@@ -8,7 +8,7 @@ from github_handling import github_handler
 
 def assign_pull_request_size(lines_changes: int) -> str:
     """Assign size of PR is label is not provided."""
-    if lines_changes > 1000:
+    if lines_changes >= 1000:
         return "XXL"
     elif lines_changes >= 500 and lines_changes <= 999:
         return "XL"

